src/toggle-switch/toggle-switch.scss

```diff
@@ -2,194 +2,206 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 
+$toggle-switch-height: $iui-baseline * 2;
+
 @mixin iui-toggle-switch {
   @include iui-reset;
-  display: flex;
+  display: grid;
+  grid-template: 'toggle';
+  grid-column-gap: $iui-s;
   align-items: center;
   font-size: $iui-font-size;
   width: fit-content;
-  padding: ($iui-baseline/2) 0;
+  padding: ($iui-baseline * 0.5) 0;
   user-select: none;
   cursor: pointer;
   position: relative;
+  isolation: isolate;
   @include themed {
     color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-2));
   }
 
-  > input {
-    width: 0;
-    height: 0;
+  > input[type='checkbox'] {
+    @include iui-reset;
+    @include iui-focus($thickness: 1px, $offset: 1px);
+    grid-area: toggle;
+    display: inline-block;
+    flex-shrink: 0;
+    position: relative;
     appearance: none;
```

Comment:
This wont work in IE.
![image](https://user-images.githubusercontent.com/9084735/152397361-3ac4db13-9889-4136-bb9e-a7e5296cb7b4.png)

Maybe that's fine though, because it is functional at least.

Reply:
You know my position about IE support 😀

Reply:
I made some changes so IE should display a normal checkbox but all other modern browsers should get the full toggle.  Can someone with IE confirm my latest push?

Reply:
Checkbox is good, grid needs to be fixed
![image](https://user-images.githubusercontent.com/9084735/152583638-9549850f-ef8b-4014-b004-a9c068d0905e.png)

I was hoping autoprefixer would handle that automatically

Reply:
Wtf guys? How it is ok? It is regression. We haven't deprecated IE yet...

Reply:
I think I have an idea of how to fix this.  I'm leaving this comment here for myself on Monday so I can try it then:
I'm going to use `::before` for the entire body of the toggle (the background color & border) instead of just the border.  I will use z-index to place them above the unstyled checkbox that appears in IE.

Reply:
It's ok to not isolate the z-index in IE as long as it works. Isolation is for preventing side effects when the component interacts with other elements beside it. If you only use positive z-indexes it should be fine

Reply:
I put the following code after the `input` selector, and it seemed to work for me

```scss
&::-ms-check {
    display:none;
 }
```

![image](https://user-images.githubusercontent.com/78564221/154345272-f0771485-a721-46d6-afc1-9aa9b9602c1c.png)

Reply:
[Reverted back to what is shown in this post](https://github.com/iTwin/iTwinUI/pull/465#discussion_r799715004).  I tried getting this to display correctly in IE a couple of ways and it doesn't seem feasible without going back to our old method of `input ~ .iui-toggle`.